lib/mgmt_msg_native.h

```diff
@@ -233,16 +233,30 @@ _Static_assert(sizeof(struct mgmt_msg_tree_data) ==
 		       offsetof(struct mgmt_msg_tree_data, result),
 	       "Size mismatch");
 
+/**
+ * enum mgmt_msg_get_data_content - content type for get_data request.
+ * @GET_CONTENT_STATE: return only "config false" data.
+ * @GET_CONTENT_CONFIG: return only "config true" data.
+ * @GET_CONTENT_ALL: return all data.
+ */
+enum mgmt_msg_get_data_content {
+	GET_CONTENT_STATE,
+	GET_CONTENT_CONFIG,
+	GET_CONTENT_ALL,
```

Comment:
I think this is a bit more natural as flags, so `GET_DATA_FLAG_STATE = 0x1`, `GET_DATA_FLAG_CONFIG = 0x2`. This allows for `CHECK_FLAGS(content, GET_DATA_FLAG_STATE)` for both the STATE and the ALL case, for example.

Reply:
Done. I changed the "exact" field to a flag as well.